client/apps/Planning/PlanningListSubNav.tsx

```diff
@@ -22,10 +23,12 @@ interface IProps {
     currentInterval: 'DAY' | 'WEEK' | 'MONTH';
     sortOrder: SORT_ORDER;
     sortField: SORT_FIELD;
-
+    users: Array<IUser>
     setStartFilter(value?: moment.Moment): void;
     jumpTo(interval: 'TODAY' | 'BACK' | 'FORWARD'): void;
     setJumpInterval(interval: 'DAY' | 'WEEK' | 'MONTH'): void;
+    activefilter: PLANNING_VIEW;
+    coverageUser : string;
```

Comment:
As this param may be `undefined`, it should be optional in the props.
It would also be more meaningful to use `IUser['_id']` here so it describes what type of data it is and the context